modules/openapi-generator/src/main/resources/python/api.mustache

```diff
@@ -35,307 +40,237 @@ class {{classname}}:
+    {{#asyncio}}async {{/asyncio}}def {{operationId}}{{>partial_api_args}} -> {{{returnType}}}{{^returnType}}None{{/returnType}}:
+{{>partial_api}}
+        response_data = {{#asyncio}}await {{/asyncio}}self.api_client.call_api(
+            *_param,
+            _request_timeout=_request_timeout
+        )
+        {{#asyncio}}await {{/asyncio}}response_data.read()
+        return self.api_client.response_deserialize(
+            response_data=response_data,
+            response_types_map=_response_types_map,
+        ).data
 
-{{#notes}}
-        {{{.}}}  # noqa: E501
-{{/notes}}
-{{^asyncio}}
-        This method makes a synchronous HTTP request by default. To make an
-        asynchronous HTTP request, please pass async_req=True
 
-        >>> thread = api.{{operationId}}({{#allParams}}{{paramName}}, {{/allParams}}async_req=True)
-        >>> result = thread.get()
-{{/asyncio}}
+    @validate_call
+    {{#asyncio}}async {{/asyncio}}def {{operationId}}_with_http_info{{>partial_api_args}} -> ApiResponse[{{{returnType}}}{{^returnType}}None{{/returnType}}]:
+{{>partial_api}}
+        response_data = {{#asyncio}}await {{/asyncio}}self.api_client.call_api(
+            *_param,
+            _request_timeout=_request_timeout
+        )
+        {{#asyncio}}await {{/asyncio}}response_data.read()
+        return self.api_client.response_deserialize(
+            response_data=response_data,
+            response_types_map=_response_types_map,
+        )
 
-{{#allParams}}
-        :param {{paramName}}:{{#description}} {{{.}}}{{/description}}{{#required}} (required){{/required}}{{#optional}}(optional){{/optional}}
-        :type {{paramName}}: {{dataType}}{{#optional}}, optional{{/optional}}
-{{/allParams}}
-{{^asyncio}}
-        :param async_req: Whether to execute the request asynchronously.
-        :type async_req: bool, optional
-{{/asyncio}}
-        :param _request_timeout: timeout setting for this request.
-               If one number provided, it will be total request
-               timeout. It can also be a pair (tuple) of
-               (connection, read) timeouts.
-        :return: Returns the result object.
-                 If the method is called asynchronously,
-                 returns the request thread.
-        :rtype: {{returnType}}{{^returnType}}None{{/returnType}}
-        """
-        kwargs['_return_http_data_only'] = True
-        if '_preload_content' in kwargs:
-            message = "Error! Please call the {{operationId}}_with_http_info method with `_preload_content` instead and obtain raw data from ApiResponse.raw_data"  # noqa: E501
-            raise ValueError(message)
-
-        return {{#asyncio}}await {{/asyncio}}self.{{operationId}}_with_http_info.raw_function(
-            {{#allParams}}
-            {{paramName}},
-            {{/allParams}}
-            **kwargs,
+
+    @validate_call
+    {{#asyncio}}async {{/asyncio}}def {{operationId}}_without_preload_content{{>partial_api_args}} -> RESTResponseType:
```

Comment:
Instead of `_without_preload_content`, this could also be called `_raw` to be shorter. 

Just a suggestion. If you go with it, best make sure name clashes are handled, because I find it sort of likely that there are REST APIs that have both a `get` and a `get_raw` endpoint.

Reply:
I can't think of a way to handle name conflicts, any ideas?

Reply:
Maybe you could export the suffix to a parameter upon generation? That way it would work for most use cases but if someone happens to have conflicts they can just choose a different one.

Reply:
> Instead of `_without_preload_content`, this could also be called `_raw` to be shorter.
> 
> Just a suggestion. If you go with it, best make sure name clashes are handled, because I find it sort of likely that there are REST APIs that have both a `get` and a `get_raw` endpoint.

If the OpenAPI method is called get_raw, the generated method would be called get_raw_raw, no?

Reply:
For an OpenAPI method get, the raw endpoint would be called get_raw and clash with the OpenAPI method get_raw.

I would like it best if the generator automatically added a trailing underscore to make sure the raw function name is unique. Maybe we should discuss this in a separate issue though, it could be solved after this PR is merged.

Reply:
I see. It's already the case with the previous functions though, right? (One could have defined q `get_with_http_info| OpenAPI method, and it would have clashed too)

Why don't we keep it simple here, with the "long" name?
The preload content feature was hidden before behind a very special flag, I would assume it wasn't really used that much by external callers and mostly used internally, in which case:

1. The long name is fine: internal calls don't care, external users are probably going to use it seldomly anyway.
2. I'm not even sure if this would be actually used at all.

So I would say: keep the current long name for now.